src/admin-views/editor/fieldset/price.php

```diff
@@ -103,19 +109,19 @@ class="ticket_field ticket_form_right"
 			$sale_price_label = esc_html__( 'Sale/Member Price:', 'event-tickets' );
 			$sale_price_desc  = esc_html__( 'Current sale or member price. This can be managed via the product editor.', 'event-tickets' );
 		}
-	?>
-	<div class="input_block">
-		<label for="ticket_sale_price" class="ticket_form_label ticket_form_left"><?php echo $sale_price_label; ?></label>
-		<input
-			type="text"
-			id="ticket_sale_price"
-			name='ticket_sale_price'
-			class="ticket_field ticket_form_right"
-			size="7"
-			value="<?php echo esc_attr( $ticket ? $sale_price : null ); ?>"
-			readonly
-		/>
-		<p class="description ticket_form_right"><?php echo $sale_price_desc; ?></p>
-	</div>
+		?>
+		<div class="input_block">
+			<label for="ticket_sale_price" class="ticket_form_label ticket_form_left"><?php echo $sale_price_label; ?></label>
+			<input
+				type="text"
+				id="ticket_sale_price"
+				name='ticket_sale_price'
+				class="ticket_field ticket_form_right"
+				size="7"
+				value="<?php echo esc_attr( $sale_price ); ?>"
+				readonly
+			/>
+			<p class="description ticket_form_right"><?php echo $sale_price_desc; ?></p>
```

Comment:
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$sale_price_desc'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$sale_price_desc'.

<sub>hash: 006d5bbc28bdabea7dd21f371c4e3650</sub>

Reply:
fixed